playbooks/app_dev/kdl.adoc

```diff
@@ -0,0 +1,174 @@
+---
+---
+= KDL a notation to describe Kubernetes application deployments
+Raffaele Spazzoli <rspazzol@redhat.com>
+v1.0, 2017-08-28
+:toc: macro
+:toc-title:
+
+toc::[]
+
+
+
+== Introduction
+
+This blog posts illustrates a graphical notation for Kubernetes API objects: Kubernetes Deployment Language (in short KDL). Kubernetes API objects can be used to describe how a solution will be deployed in Kubernetes.
+
+I think there is a need to describe and document how applications will be deployed in Kubernetes, especially when these applications are comprised of several components. 
+I wanted to create a simple graphical convention to describe these deployments, so that these diagram could be easily whiteboard and the capture in a document.
```

Comment:
To fill this gap, this document defines a simple graphical convention to describe these deployments, so that they can be diagramed, easily whiteboarded, and captured in a document.

Reply:
@raffaelespazzoli First person voice works in a blog post but is sometimes looked down upon in more technical writing (though I generally prefer it), such as a Spec or an RFC (which I think is the aim here).  @etsauer How do you feel about that?  I'm willing to back off on this and the comment above if you feel it is ok here.

Reply:
@lpsantil Agreed. made the change